app/views/course/dashboards/show.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1 class="mb-3">Hi, <%= current_user.email %></h1>
-<%= link_to "Learn", course_section_path(@section), class: "btn btn-primary" %>
+<h1 class="mb-3">Welcome Back!</h1>
+
+<h2>Time For A Review</h2>
+<p>Here are your missed questions for the last three days. Spend some time reviewing them before retaking the quiz.</p>
+<div class="mb-3">
+  <ol>
+    <% current_user.answers.wrong.each do |answer| %>
+      <% if answer.quiz_questions.last.created_at > 3.days.ago %>
```

Comment:
There's an issue with those 2 lines.

Quick explanation:
`Line 7` will get ALL wrong answers of the user from the DB, instantiate ALL of them, but `Line 8` is only using a few.

`created_at > 3.days.ago` should be a condition on line 7, making sure only the required answers are selected and instantiated from the DB.

It's not a huge performance issue currently, but it's a good thing to learn and avoid early on.

Reply:
Also, we should actually use the `updated_at` column instead of `created_at` I realized that he quiz question is created along with the quiz, so the quiz may be 10 days old (along with the user answers for it), but only have been answered yesterday.

So, my suggestion, using a scope for answers;
```ruby
class Answer < ApplicationRecord
  # ...
  scope :recently_answered, -> { where("quiz_questions.updated_at > ?", 3.days.ago) }
  # ...
end
```

Reply:
So line 7 can be:
```ruby
current_user.answers.recently_answered.wrong.each
```

And we do away with the conditional set on line 8.

Reply:
The requested changes are resolved in the last two commits and the branch is ready for review again.